### PR TITLE
fix(a380x/fcu): altitude and baro instruments should show values during light test

### PR DESCRIPTION
**Pull request by mattgogerly**
Fixes https://github.com/flybywiresim/aircraft/issues/9092

## Summary of Changes

The OverheadPublisher was not being instantiated anywhere, so the subscription in the Baro/Altitude components was never receiving any events. Now they should show the test values as expected.

## Screenshots (if necessary)

Pending model release...

## References

See issue for pictures

## Additional context

Discord username (if different from GitHub): nothematic

## Testing instructions

1. Load into the sim & power on
2. Press light test switch
3. Altitude should show "88888" and baro should show "8.8.8.8"

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

**Review by tracernz — APPROVED** [review]

**Comment by alepouna** [comment]
Needs changelog conflict fix and lgtm

**Comment by 2hwk** [comment]
You should get the contributor role on our discord shortly, thank you for your first contribution to the repository.

**Comment by mattgogerly** [comment]
> You should get the contributor role on our discord shortly, thank you for your first contribution to the repository.

Do you want me to fix the changelog conflict or will a bot magically do it?

**Comment by alepouna** [comment]
> > You should get the contributor role on our discord shortly, thank you for your first contribution to the repository.
> 
> Do you want me to fix the changelog conflict or will a bot magically do it?

No bot yet to handle that, needs to be done by you! 🙌

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 ## 0.13.0
 
 1. [GENERAL] Fixed issue in C++ WASM Framework that caused performance degradation in some WASM modules - @frankkopp (Frank Kopp)
+1. [FCU] QNH and altitude displays now correctly show "8" values during light test - @mattgogerly (Matt
+1. [FCU] Fixed auto-initialisation of baro unit - @tracernz (Mike)
```
Comment (alepouna) [inline .github/CHANGELOG.md:12]:
```suggestion
```
Duplicate

**Review by alepouna — CHANGES_REQUESTED** [review]

**Review by alepouna — APPROVED** [review]

**Pull request merged**